src/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -83,7 +82,7 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->defaultValue('0.0.0')
                 ->end()
                 ->booleanNode('show_webby')->defaultTrue()->info('If true, show Webby on the documentation page')->end()
-                ->booleanNode('use_symfony_listeners')->defaultFalse()->info(sprintf('Uses Symfony event listeners instead of the %s.', MainController::class))->end()
+                ->booleanNode('use_symfony_listeners')->defaultFalse()->info(\sprintf('Uses Symfony event listeners instead of the %s.', MainController::class))->end()
```

Comment:
```suggestion
                ->booleanNode('use_symfony_listeners')->defaultFalse()->info(sprintf('Uses Symfony event listeners instead of the %s.', MainController::class))->end()
```